packages/samples/headless-ssr/src/app/common/result-list.tsx

```diff
@@ -0,0 +1,17 @@
+import {Result} from '@coveo/headless';
```

Comment:
unfortunately, I think we'll eventually need to export "*Methods" types from either Headless or the HeadlessReact package.

For example:
```ts
export type ResultListMethods = Omit<ResultList, 'state' | 'subscribe'>;
```

Reply:
To be discussed of course (not for this PR).